tests/acceptance/features/bootstrap/FeatureContext.php

```diff
@@ -221,13 +186,6 @@ public function setOCSelector(string $selector): void {
 		$this->oCSelector = $selector;
 	}
 
-	/**
-	 * @return string
-	 */
-	public function getOCSelector(): string {
```

Comment:
I don't know the state of oc10-ocis parallel setup. But let's not delete these code